app/templates/people/person/positions/position/index.hbs

```diff
@@ -0,0 +1,259 @@
+                  <div class="au-o-grid__item au-u-1-4">
+                    <p><strong>Start mandaat</strong></p>
+                  </div>
+                  <div class="au-o-grid__item au-u-3-4">
+                    <p>{{date-format @model.mandatory.startDate}}</p>
+                  </div>
+                </div>
+              </div>
+              <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
+                <div class="au-o-grid">
+                  <div class="au-o-grid__item au-u-1-4">
+                    <p><strong>Bestuursperiode</strong></p>
+                  </div>
+                  <div class="au-o-grid__item au-u-3-4">
+                    <p>{{@model.mandatory.mandate.governingBody.period}}</p>
+                  </div>
+                </div>
+              </div>
+              <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
+                <div class="au-o-grid">
+                  <div class="au-o-grid__item au-u-1-4">
+                    <p><strong>Geplande einddatum</strong></p>
+                  </div>
+                  <div class="au-o-grid__item au-u-3-4">
+                    <p>
+                      {{date-format @model.mandatory.expectedEndDate}}
+                    </p>
+                  </div>
+                </div>
+              </div>
+              {{#if @model.mandatory.typeHalf.label}}
+                <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
+                  <div class="au-o-grid">
+                    <div class="au-o-grid__item au-u-1-4">
+                      <p><strong>Verkozen</strong></p>
+                    </div>
+                    <div class="au-o-grid__item au-u-3-4">
+                      <p>
+                        {{@model.mandatory.typeHalf.label}}
+                      </p>
+                    </div>
+                  </div>
+                </div>
+              {{/if}}
+              {{#if @model.mandatory.endDate}}
+                <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
+                  <div class="au-o-grid">
+                    <div class="au-o-grid__item au-u-1-4">
+                      <p><strong>Einddatum</strong></p>
+                    </div>
+                    <div class="au-o-grid__item au-u-3-4">
+                      <p>
+                        {{date-format @model.mandatory.endDate}}
+                      </p>
+                    </div>
+                  </div>
+                </div>
+              {{/if}}
+              {{#if @model.mandatory.reasonStopped}}
+                <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
+                  <div class="au-o-grid">
+                    <div class="au-o-grid__item au-u-1-4">
+                      <p><strong>Stopzetting</strong></p>
+                    </div>
+                    <div class="au-o-grid__item au-u-3-4">
+                      <p>
+                        {{@model.mandatory.typeHalf.label}}
```

Comment:
This uses a different attribute as in the `if` check, is that intended?

Reply:
ops